packages/react-components/review/react-components.api.md

```diff
@@ -358,14 +358,12 @@ export interface JumpToNewMessageButtonProps {
 // @public
 export const lightTheme: PartialTheme & CallingTheme;
 
-// @public
-export const LocalizationProvider: (props: LocalizationProviderProps) => JSX.Element;
-
-// @public
-export type LocalizationProviderProps = {
-    locale: ComponentLocale;
+// @public (undocumented)
+export const LocalizationProvider: (props: {
+    defaultLocale?: string | undefined;
+    locale: ComponentLocale | ((locale?: string | undefined) => Promise<ComponentLocale>);
```

Comment:
As discussed IRL, let's simplify the async loader to `() => Promise<ComponentLocale>` etc. We can also delete `useSetLocale` then.

Reply:
Thanks for the suggestion. Done.